src/js/index.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable global-require */
 var preloadWebComponents = require('./preloadWebComponents'),
-    createChromecastButton = require('./components/ChromecastButton'),
-    createChromecastTech = require('./tech/ChromecastTech'),
+    { default: createChromecastButton } = require('./components/ChromecastButton'),
```

Comment:
Removing the mix of ESM and CommonJS will mean we won't have to do `{ default: createChromecastButton }` either.

Reply:
See the comment above.